### DISCUSSION TITLE
Only the GFPGAN version images are stored in the Output folder, how i can get both versions saved?

### DISCUSSION THREAD

**Discussion by ZeroCool22** [hlky/stable-diffusion-webui#657] [Q&A] [answered]
In the previous version, we get the 2 version of the images in the OP folder:

![Both versions](https://user-images.githubusercontent.com/13344308/188348055-dd4e7504-c019-4f0d-898f-c3e6157da45c.jpg)

But now we **only** get the GFPGAN:

![Screenshot_2](https://user-images.githubusercontent.com/13344308/188348090-0059c34b-05e6-4f64-9ffb-1b7e79e619d7.jpg)

It's how works now or the non GFPGAN version are stored in other folder?

**PD:** If someone knows if i can modify the code of some file to get **both versions saved** in the output folder, would be great!

**Comment by ZeroCool22** [comment, marked as answer]
Thx to [jjisnow](https://github.com/jjisnow) for the solution.

Switch Skip_Save to False..

**Comment by dmt-hub** [comment]
where do we find this option?

**Reply by ZeroCool22** [reply]
> where do we find this option?

webui.py in folder scripts.

**Reply by dmt-hub** [reply]
thank you
